packages/vm/lib/state/stateManager.ts

```diff
@@ -111,7 +115,7 @@ export default class DefaultStateManager implements StateManager {
     const codeHash = keccak256(value)
 
     if (codeHash.equals(KECCAK256_NULL)) {
-      return
+      //return
```

Comment:
just a note this is commented out

Reply:
It should not be commented out, this is a remainder from toying around in order to resolve those failing tests. It will not change anything though; the only call is from `evm.ts` which does not call `putContractCode` if the code to put is the empty string.